Formatting fixes (#105)
bcgov/bcdg: docs/use-github-in-bcgov/bc-government-organizations-in-github.md

The [Digital Principles for B.C. Government](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) supports and encourages product teams to work in the open. GitHub is the leading platform for open-source projects and collaboration. 

With GiHub Enterprise Cloud, we can:

  - Collaborate with the open-source community to build software
  - Support innvation
  - Share code and best practices 
  - Track issues and feature requests
  - Use tools for team and project management
  - Save costs and time

> The **Developer Experience team** manages B.C. Government's GitHub environments. For questions, concerns, or feedback, please [submit a support request on our Jira Service Manager system]( https://citz-do.atlassian.net/servicedesk/customer/portal/2).

Reviewer: ```suggestion
The **Developer Experience team** manages B.C. government's GitHub environments. For questions, concerns, or feedback, please [submit a support request on our Jira Service Manager system]( https://citz-do.atlassian.net/servicedesk/customer/portal/2).
```

**`bcgov-c`**: cost for security features because GH charges a fee for private repos. **But** the Developer Experience team covers cost **without cost recovery**.

#### Security Insights for GitHub Enterprise-linked organizations

> All orgs linked to the corporate GitHub Enterprise accounts have access to the [Security Insight feature](https://docs.github.com/en/enterprise-cloud@latest/code-security/how-tos/view-and-interpret-data/analyze-organization-data/viewing-security-insights), inclusive of branch protection and code owners for private repos.

Reviewer: What do you mean by orgs? Is there a way they can be described or a link to them?

Author: Changed to explicitly state bcgov and bcgov-c orgs

> If the instructions and troubleshooting help below doesn't resolve any issues, please [submit a support request on our Jira Service Manager system]( https://citz-do.atlassian.net/servicedesk/customer/portal/2).

Reviewer: ```suggestion
If the instructions and troubleshooting guide below doesn't resolve your encountered issue(s), please [submit a support request on our Jira Service Manager system]( https://citz-do.atlassian.net/servicedesk/customer/portal/2).
```
